packages/containers/containers.2.0+alpha2/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+maintainer: "simon.cruanes.2007@m4x.org"
+authors: "Simon Cruanes"
+homepage: "https://github.com/c-cube/ocaml-containers/"
+bug-reports: "https://github.com/c-cube/ocaml-containers/issues/"
+doc: "https://c-cube.github.io/ocaml-containers"
+tags: ["stdlib" "containers" "iterators" "list" "heap" "queue"]
+dev-repo: "https://github.com/c-cube/ocaml-containers.git"
+build: ["jbuilder" "build" "-p" name "-j" jobs]
+build-test: ["jbuilder" "runtest"]
```

Comment:
Don't forget `"-p" name "-j" job` for build-test as well.

Reply:
yeah, that'll be in the next release, it's really a detail.

Reply:
tsk tsk..